flow/display_list_canvas_unittests.cc

```diff
@@ -1512,8 +1518,9 @@ class CanvasCompareTester {
                    [=](SkCanvas* c, SkPaint&) { c->skew(0.05, 0.05); },
                    [=](DisplayListBuilder& b) { b.skew(0.05, 0.05); }));
     {
-      SkMatrix tx = SkMatrix::MakeAll(1.10, 0.10, 5,   //
-                                      0.05, 1.05, 10,  //
+      SkScalar tweak = 1.0 / 16.0;
```

Comment:
Please add a comment explaining this

Reply:
Just making sure my fractions are powers of 2 (0.1 and 0.05 are not).

Reply:
This reduced, but didn't eliminate the off-by-1-pixel errors I was seeing. In the end, it got it down to 1 each for the 4 primitives that still rasterized slightly differently.

Reply:
Sure - just help future us when we're reading this test to know why this is being done and what it helps address :)